visualization/app/codeCharta/ui/metricChooser/metricChooser.area.component.html

```diff
@@ -10,15 +10,18 @@
 		md-container-class="ribbonBarDropdown"
 	>
 		<md-select-header>
-			<input
-				id="area-metric-selector"
-				class="metric-search area-metric"
-				ng-model="$ctrl._viewModel.searchTerm"
-				type="text"
-				placeholder="&#xF002; Metric (highest value)"
-				ng-keydown="$event.stopPropagation()"
-				ng-change="$ctrl.filterMetricData()"
-			/>
+			<div>
```

Comment:
Classless div, is it needed here? Putting all the childs of this div directly into md-select-header would have the same result